package.json

```diff
@@ -546,6 +546,7 @@
                         "type": "string",
                         "enum": [
                             "~1",
+                            "~2",
                             "beta"
```

Comment:
beta's in the enum here, but it's missing from runtime.md and api.md and elsewhere...?

Reply:
Yeah a lot of people will have it in their settings for VS Code. I figure we leave it for a release or two before removing it. Added descriptions so that it will now say "Deprecated":

![screen shot 2018-09-12 at 1 32 11 pm](https://user-images.githubusercontent.com/11282622/45451788-fd899c00-b690-11e8-9b89-5160663614c2.png)